apps/src/applab/debugArea.js

```diff
@@ -0,0 +1,236 @@
+/* jshint
+ funcscope: true,
+ newcap: true,
+ nonew: true,
+ shadow: false,
+ unused: true,
+
+ maxlen: 90,
+ maxparams: 3,
+ maxstatements: 200
+ */
+/* global $ */
+
+var applabMsg = require('./locale');
+var dom = require('../dom');
+var utils = require('../utils');
+
+var debugArea = module.exports;
+
+/** @type {jQuery} */
+var rootDiv_ = null;
```

Comment:
I thought about it and ventured down that path a bit, but then saw it was contrary to the current design of applab.  If you're in favor of actually making this a class, I'll be happy to adjust it.

The hardest part of my first attempt to extract a debug area controller was some entangled responsibility, and the debug controls needing to reach back into applab to control the interpreter.  There's just a pretty good amount of wiring to get all the hooks into the debug area in a clean way.  For this change I'm only moving the new show/hide functionality into the controller so it's less messy.